### PR TITLE
feature(yf): add README.md for the repo

### PR DESCRIPTION
**Pull request by VaninaY**
I am creating the README file. However I don't know how to add pic and ref (I will do it later).

### CODE REVIEW

**Comment by VaninaY** [comment]
pics added

**Comment by VaninaY** [comment]
hypelink for reference added

File: README.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+2. A problem of protein structure completion problem for a given partial structure, such as the recent Science [1] by the famous David Baker group. This assumes that only partial structural matches can be found. 
+3. Combination of the fitted energy function with MD simulation for protein design, such as the recent Nature [2] of Liu Haiyan's team in China.
+
+In addition, many methods can be used for protein design, and the corresponding AI problem definitions are also very different. This article lists some high-level articles in AI -based protein design, which will be continuously updated in the future.
+
+
+## Papers
+
+```
+format:
```
Comment (huyingjun1) [inline README.md:42]: 这部分是准备后续补充吗？

File: README.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+- [Conditional Antibody Design as 3D Equivariant Graph Translation](https://arxiv.org/abs/2208.06073)
+  - Xiangzhe Kong, Wenbing Huang, Yang Liu
+  - Keywords: antibody design, graph tranlation
+  
+- [Deep sharpening of topological features for de novo protein design](https://openreview.net/forum?id=DwN81YIXGQP)
+  - Zander Harteveld, Joshua Southern, Michaël Defferrard, Andreas Loukas, Pierre Vandergheynst, Micheal Bronstein, Bruno Correia
+  - Keywords: topological feature, protein design
+  
+- [Function-guided protein design by deep manifold sampling](Function-guided protein design by deep manifold sampling)
```
Comment (huyingjun1) [inline README.md]: 缺少超链接？

File: README.md [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+  - [ICML, ICLR or NeurIPS 2022](#icml-iclr-or-neurips-2022)
+  - [Arxiv or bioRxi](#arxiv-or-biorxi)
+  - [Others](#others)
+- [Contributing](#contributing)
+
+## Overview of Protein Design
+
+AI tools have solved the protein structure prediction problem. This problem derives the spatial structure from the amino acid sequence and achieves atomic-level prediction accuracy, such as Alphafold 2. It combines previous protein structure prediction models to automatically learn protein design methods, thus truly serving human pharmaceutical needs.
+
+![image info1](./alphafold.png)
```
Comment (huyingjun1) [inline README.md]: 图片无法显示 不确定是否为个人问题 建议检查路径

File: README.md (hunk 1 above)
Comment (huyingjun1) [reply README.md:42]: 亦或者是下面论文展示的格式？

Comment (VaninaY) [reply README.md:42]: 是下面论文展示的格式的template

File: README.md (hunk 2 above)
Comment (VaninaY) [reply README.md]: 原文就没有，没注意 现在补上

File: README.md (hunk 3 above)
Comment (VaninaY) [reply README.md]: 是图片的pr也没merge的问题应该

File: README.md (hunk 2 above)
Comment (VaninaY) [reply README.md]: 已完成补充。

File: README.md [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+## Table of Contents
+
+- [A Overview / Example of AI based Protein Design](#a-Overview/Example-of-AI-based-Protein-Design)
+- [Papers](#papers)
+
+  - [Nature](#nature) (**<font color="red">Important!!!</font>**)
+  - [Nature Biomedical Engineering](#nature-biomedical-engineering)
+  - [Nature Communications](#nature-communications)
+  - [Science](#science)
```
Comment (TuTuHuss) [inline README.md:18]: maybe ''science''  's ranking is higher than  ''nature biomedical engineering'' and ''nature communicatons'' ?

File: README.md [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+
+[2] Huang, Bin, et al. ["A backbone-centred energy function of neural networks for protein design." Nature 602.7897 (2022): 523-528. ](https://www.nature.com/articles/s41586-021-04383-5)
+
+## Contributing
+
+Our purpose is to make this repo even better. If you are interested in contributing, please refer to [HERE](CONTRIBUTING.md) for instructions in contribution.
+
+## License
+
+Awesome Decision Transformer is released under the Apache 2.0 license.
```
Comment (TuTuHuss) [inline README.md]: decision transformer?  lol

Comment (VaninaY) [reply README.md]: 哈哈哈哈哈哈我偷偷从胤民那复制来的

File: README.md [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+- [Robust deep learning based protein sequence design using ProteinMPNN](https://www.science.org/doi/abs/10.1126/science.add2187)
+  -  J. Dauparas,  I. Anishchenko,  N. Bennett,  H. Bai,  R. J. Ragotte,  L. F. Milles,  B. I. M. Wicky,  A. Courbet, R. J. de Haas,  N. Bethel,  P. J. Y. Leung,  T. F. Huddy,  S. Pellock,  D. Tischer,  F. Chan,  B. Koepnick, H. Nguyen, A. Kang,  B. Sankaran,  A. K. Bera, N. P. King,  D. Baker
+  - Keywords: language model, structure prediction
+  
+- [Scaffolding protein functional sites using deep learning](https://www.science.org/doi/abs/10.1126/science.abn2100)
+  -  Jue Wang, Sidney Lisanza, David Juergens, Doug Tischer, Joseph L. Watson, Karla M. Castro, Robert Ragotte, Amijai Saragovi, Lukas F. Milles, Minkyung Baek, Ivan Anishchenko, Wei Yang, Derrick R. Hicks, Marc Expòsit, Thomas Schlichthaerle, Jung-Ho Chun, Justas Dauparas, Nathaniel Bennett, Basile I. M. Wicky, Andrew Muenks, Frank DiMaio, Bruno Correia, Sergey Ovchinnikov, David Baker
+  - Keywords: functional site, deep learning, hallucination, inpainting
+  
+
+### ICML, ICLR or NeurIPS
```
Comment (TuTuHuss) [inline README.md]: line19 states that only 2022 will be included？

File: README.md (hunk 4 above)
Comment (VaninaY) [reply README.md:18]: nature xx is sub-journal of nature

File: README.md (hunk 6 above)
Comment (VaninaY) [reply README.md]: true, I will add the time for it

File: README.md (hunk 4 above)
Comment (TuTuHuss) [reply README.md:18]: i know, if sub-journal of nature  has more influence than science ，then i have no problem

File: README.md [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+
+- [A Overview / Example of AI based Protein Design](#a-Overview/Example-of-AI-based-Protein-Design)
+- [Papers](#papers)
+
+  - [Nature](#nature) (**<font color="red">Important!!!</font>**)
+  - [Nature Biomedical Engineering](#nature-biomedical-engineering)
+  - [Nature Communications](#nature-communications)
+  - [Science](#science)
+  - [ICML, ICLR or NeurIPS 2022](#icml-iclr-or-neurips-2022)
+  - [Arxiv or bioRxi](#arxiv-or-biorxi)
```
Comment (PaParaZz1) [inline README.md]: bioRxiv？

File: README.md [hunk 8]
```diff
@@ -0,0 +1,184 @@
...
+  - [Others](#others)
+- [Contributing](#contributing)
+
+## Overview of Protein Design
+
+AI tools have solved the protein structure prediction problem. This problem derives the spatial structure from the amino acid sequence and achieves atomic-level prediction accuracy, such as Alphafold 2. It combines previous protein structure prediction models to automatically learn protein design methods, thus truly serving human pharmaceutical needs.
```
Comment (PaParaZz1) [inline README.md]: AlphaFold

File: README.md [hunk 9]
```diff
@@ -0,0 +1,184 @@
...
+
+## Papers
+
+```
+format:
+- [title](paper link) [links]
+  - author1, author2, and author3...
+  - publisher
+  - keyword
+  - code
```
Comment (PaParaZz1) [inline README.md]: remove code and env item

File: README.md [hunk 10]
```diff
@@ -0,0 +1,184 @@
...
+  
+## Reference
+
+[1] Wang, Jue, et al. ["Scaffolding protein functional sites using deep learning." Science 377.6604 (2022): 387-394.](https://www.science.org/doi/abs/10.1126/science.abn2100)
+
+[2] Huang, Bin, et al. ["A backbone-centred energy function of neural networks for protein design." Nature 602.7897 (2022): 523-528. ](https://www.nature.com/articles/s41586-021-04383-5)
+
+## Contributing
+
+Our purpose is to make this repo even better. If you are interested in contributing, please refer to [HERE](CONTRIBUTING.md) for instructions in contribution.
```
Comment (PaParaZz1) [inline README.md:176]: where is `CONTRIBUTING.md`

File: README.md [hunk 11]
```diff
@@ -0,0 +1,184 @@
...
+
+## License
+
+Awesome AI based Protein Design is released under the Apache 2.0 license.
```
Comment (PaParaZz1) [inline README.md]: awesome AI based protein design

File: README.md (hunk 8 above)
Comment (PaParaZz1) [inline README.md]: add link for AlphaFold 2

**Review by PaParaZz1 — CHANGES_REQUESTED** [review]

File: README.md (hunk 10 above)
Comment (VaninaY) [reply README.md:176]: added in PR#2

**Review by PaParaZz1 — APPROVED** [review]

**Pull request merged**